nondex-test/pom.xml

```diff
@@ -108,5 +108,12 @@
       <artifactId>nondex-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
+      <version>4.1</version>
+      <scope>system</scope>
+      <systemPath>/Users/kaku/Desktop/commons-collections4-4.1.jar</systemPath>
```

Comment:
If you do not give a system path, it will download the version you specify. In this case your version is 4.1, so it will download that. If you want 4.4.1 you should specify that version.